tests/bdd/scale/test_NAS_T1123.py

```diff
@@ -0,0 +1,142 @@
+# coding=utf-8
+"""SCALE UI: feature tests."""
+
+import time
+from function import (
+    wait_on_element,
+    is_element_present,
+    attribute_value_exist,
+    run_cmd,
+    ssh_cmd,
+    post
+)
+from pytest_bdd import (
+    given,
+    scenario,
+    then,
+    when,
+    parsers
+)
+
+
+@scenario('features/NAS-T1123.feature', 'Create an smb share with the system ACL dataset')
+def test_create_an_smb_share_with_the_system_acl_dataset():
+    """Create an smb share with the system ACL dataset."""
+
+
+@given('the browser is open, the FreeNAS URL and logged in')
+def the_browser_is_open_the_freenas_url_and_logged_in(driver, nas_ip, root_password):
+    """the browser is open, the FreeNAS URL and logged in."""
+    if nas_ip not in driver.current_url:
+        driver.get(f"http://{nas_ip}")
+        assert wait_on_element(driver, 10, '//input[@data-placeholder="Username"]')
+    if not is_element_present(driver, '//mat-list-item[@ix-auto="option__Dashboard"]'):
+        assert wait_on_element(driver, 10, '//input[@data-placeholder="Username"]')
+        driver.find_element_by_xpath('//input[@data-placeholder="Username"]').clear()
+        driver.find_element_by_xpath('//input[@data-placeholder="Username"]').send_keys('root')
+        driver.find_element_by_xpath('//input[@data-placeholder="Password"]').clear()
+        driver.find_element_by_xpath('//input[@data-placeholder="Password"]').send_keys(root_password)
+        assert wait_on_element(driver, 5, '//button[@name="signin_button"]')
+        driver.find_element_by_xpath('//button[@name="signin_button"]').click()
+    else:
+        driver.find_element_by_xpath('//mat-list-item[@ix-auto="option__Dashboard"]').click()
+
+
+@when('you should be on the dashboard, click on Sharing then Windows Shares(SMB)')
+def you_should_be_on_the_dashboard_click_on_sharing_then_windows_sharessmb(driver):
+    """you should be on the dashboard, click on Sharing then Windows Shares(SMB)."""
+    time.sleep(1)
+    assert wait_on_element(driver, 10, '//mat-list-item[@ix-auto="option__Dashboard"]', 'clickable')
+    driver.find_element_by_xpath('//mat-list-item[@ix-auto="option__Dashboard"]').click()
+    time.sleep(1)
+    assert wait_on_element(driver, 10, '//span[contains(.,"Dashboard")]')
+    driver.find_element_by_xpath('//mat-list-item[@ix-auto="option__Shares"]').click()
+
+
+@then('The Windows Shares(SMB) page should open, Click Add')
+def the_windows_sharessmb_page_should_open_click_add(driver):
+    """The Windows Shares(SMB) page should open, Click Add."""
+    time.sleep(1)
```

Comment:
instead of sleep use  wait_on_element for the Sharing title.
to even make sure that the page is loaded you can add a wait_on_element Windows (SMB) Shares header